presentation/src/main/java/com/example/myapplication/EndDateButtonList.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.myapplication
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.myapplication.ui.theme.MyApplicationTheme
+import com.example.myapplication.ui.theme.SweetPotato
+import com.example.myapplication.ui.theme.White
+import com.example.presentation.R
+
+@Composable
+fun EndDateButton(
+    text: Int,
+    width: Int,
+) {
+    Box(
+        modifier = Modifier
+            .clip(RectangleShape)
+            .fillMaxHeight()
+            .width(width.dp)
+            .background(color = SweetPotato)
+            .clickable { }
```

Comment:
```suggestion
            .clickable { /*TODO*/ }
```
구현되어야 될 부분에 `TODO`를 붙이는게 좋은 습관이 될 수 있습니다!